docker/ubuntu1604/Dockerfile

```diff
@@ -52,8 +52,9 @@ RUN echo "deb http://apt.openvstorage.com unstable main" > /etc/apt/sources.list
     	    libxio0 libxio0-dbg libxio0-dbgsym \
     	    libxio-dev libxio-dev-dbgsym && \
     apt-get install -y libev4 && \
-    cd /root && apt-get download --allow-unauthenticated arakoon && \
+    cd /tmp && apt-get download --allow-unauthenticated arakoon && \
```

Comment:
No problem to use something else than /root. The problem is the jenkins builds share /tmp on the buildslave so either create a unique directory in /tmp first and use that (and cleanup further on) or use something else inside the container if /root does not work.